index.html

```diff
@@ -174,10 +174,81 @@
                 loadL10nSplashScreen();
             </script>
             <div class="loading-text" id="loadingText" style="margin-top:1.5rem;"></div>
+            <a href="https://www.sugarlabs.org/" target="_blank" id="link-to-sugarLabs">
+                <div class= "bottom-right-logo1" id="bottom-right-logo">
+                    <h1  class="upper-text" id="this-is-a-text">This is a</h1>
+                </div>     
+                <div class="logo-container" id="bottom-right-logo">
+                    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 501 167" height="50" width="150">
+                        <!-- Define the gradient for the rainbow animation in 'sugar'-->
```

Comment:
I don't like the way the colors are animated.

Perhaps it should just choose a random color, similar to our logo on "www".

I could entertain a color animation that just fades from one color to another, but without the fine gradient (a less intense animation).

Reply:
I think to start we'll just use one color.